Sample Solutions/Viva Insights Query Analysis Automation/Notebooks/vivaInsights_person.ipynb

```diff
@@ -88,7 +69,8 @@
         "from pyspark.sql.types import DateType\r\n",
         "\r\n",
         "#Reading meeting csv file from storage account\r\n",
-        "personDf = spark.read.csv(inputFilePath.format(extractionFS, StorageAccountName, PipelineId, PersonQueryDatasetFolder), header = 'true', inferSchema= 'true')\r\n",
+        "inputFilePath = 'abfss://{0}@{1}.dfs.core.windows.net/{2}/raw/{3}/*.txt'.format(VivaInsightsDataFileSystem, StorageAccountName, PipelineId, MeetingQueryDatasetFolder)\r\n",
```

Comment:
MeetingQueryDatasetFolder doesn't look to be defined?

Reply:
It is in the parameter cell on top